### ISSUE TITLE
Compilation Error on macOS 10.15 Catalina

### ISSUE THREAD

**Issue by donniemattingly** [versilov/matrex#23]
The proposed solution to the error: `native/src/matrix_dot.c:5:10: fatal error: 'cblas.h' file not found`

is 

`open /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg`

however, that directory does not exist on macOS 10.15. The solution [here](https://stackoverflow.com/questions/58278260/cant-compile-a-c-program-on-a-mac-after-upgrading-to-catalina-10-15/58349403#58349403) did work for me. 

```bash
export CPATH=/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/
```

Honestly though this is just the first working result on google, not sure if it makes sense to add to the README or not.

**Comment by versilov** [comment]
At least it's worth noting in README. Thank you!

**Comment by krisalyssa** [comment]
The solution provided by @donniemattingly did not work for me on macOS 10.15.1. Eventually I figured out that this appears to:

C_INCLUDE_PATH=/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/System/Library/Frameworks/Accelerate.framework/Frameworks/vecLib.framework/Headers mix compile

I ran a `find` in the directory named in the `CPATH` env var above, and `cblas.h` isn't anywhere in that tree. In fact, it looks like none of the Accelerate or vecLib header files are.
